rpcs3/Emu/Cell/ErrorCodes.h

```diff
@@ -51,7 +65,7 @@ enum CellNotAnError : s32
 
 // Constructor specialization that doesn't trigger reporting
 template <>
-constexpr FORCE_INLINE error_code::error_code(const CellNotAnError& value)
+constexpr FORCE_INLINE error_code::error_code(CellNotAnError&& value)
```

Comment:
Why is this? It's not a perfect forwarding context.

Reply:
Fixed.